src/ui/RepoView.cpp

```diff
@@ -2134,8 +2141,8 @@ void RepoView::popStash(int index) {
     error(entry, tr("pop stash"), commit.link());
     return;
   }
-
-  refresh(false);
+  // switch back to head
+  selectReference(mRepo.head());
```

Comment:
Sometimes after popping a stash, uncommitted changes was not selected.  This change fixes that.
```suggestion
  selectReference(mRepo.head());
  selectFirstCommit();
```